src/go/configmanager/flags/flags.go

```diff
@@ -66,6 +66,7 @@ var (
 	ServiceAccountKey = flag.String("service_account_key", "", `Use the service account key JSON file to access the service control and the
 	service management.  You can also set {creds_key} environment variable to the location of the service account credentials JSON file. If the option is
   omitted, the proxy contacts the metadata service to fetch an access token`)
+	LocalAccessTokenServerPort = flag.Uint("local_access_token_server_port", 8791, "Port that configmanager use to setup server to provide envoy with access token using service account credential, for accessing servicecontrol.")
```

Comment:
just call it token_agent_port?

Reply:
Not sure if this port is a security risk.  If a hacker can access this port, he can get access token.  But normally, we don't expose this port outside of container, it should be fine.

Reply:
This port should be treated same as discovery_port(the port of xds, 8790 by default), both of which should not be exposed.

Reply:
Should be fine. Our threat model indicates the riskiest threat is malicious clients attacking Envoy. Getting ssh access in the container / remote code execution is very low probability.